ansible/roles/stack-sunbird/templates/sunbird_program.env

```diff
@@ -16,6 +16,9 @@ telemetry_service_endpoint=v1/telemetry
 telemetry_service_host=http://telemetry-service:9001
 sunbird_kafka_host={{groups['sunbird-processing-cluster-kafka']|join(':9092,')}}:9092
 sunbird_auto_creation_topic={{ sunbird_auto_creation_topic }}
+sunbird_question_bulkupload_topic={{ env_name }}.av.question.bulkupload
+sunbird_kafka_bulkupload_consumer_group_id={{ env_name }}.av.qstn.bulkupload
+sunbird_assessment_service_base_url={{ sunbird_kp_assessment_service_base_url }}
```

Comment:
Please change this to `http://assessment-service:9000`

Reply:
@keshavprasadms changed the value as suggested and updated the PR.

Reply:
@Hari-1027 Merging this. Please note the value does not end with `/`. Your need to append it in your code.
Or else you can update the variable value to end with a `/`

Reply:
For reference - https://github.com/project-sunbird/sunbird-devops/blob/3d3335348ba61b1d4e33bb5cad2658ef8156d30e/ansible/roles/stack-sunbird/templates/sunbird_program.env#L23

Reply:
Thank you @keshavprasadms will make sure to append the '/' in our code.